src/main/java/uk/gov/pay/ledger/util/csv/FlatCsvTransaction.java

```diff
@@ -0,0 +1,245 @@
+package uk.gov.pay.ledger.util.csv;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.pay.ledger.transaction.model.CardDetails;
+import uk.gov.pay.ledger.transaction.model.CardType;
+import uk.gov.pay.ledger.transaction.search.model.TransactionView;
+import uk.gov.pay.ledger.transaction.state.ExternalTransactionState;
+
+import java.text.DecimalFormat;
+import java.time.format.DateTimeFormatter;
+import java.util.Optional;
+
+public class FlatCsvTransaction {
+
+    @JsonProperty("Reference")
+    private String reference;
+
+    @JsonProperty("Description")
+    private String description;
+
+    @JsonProperty("Email")
+    private String email;
+
+    @JsonProperty("Amount")
+    private String amount;
+
+    @JsonProperty("Card Brand")
+    private String cardBrand;
+
+    @JsonProperty("Cardholder Name")
+    private String cardholderName;
+
+    @JsonProperty("Card Expiry Date")
+    private String cardExpiryDate;
+
+    @JsonProperty("Card Number")
+    private String cardNumber;
+
+    @JsonProperty("State")
+    private String state;
+
+    @JsonProperty("Finished")
+    private Boolean finished;
+
+    @JsonProperty("Error Code")
+    private String errorCode;
+
+    @JsonProperty("Error Message")
+    private String errorMessage;
+
+    @JsonProperty("Provider ID")
+    private String providerId;
+
+    @JsonProperty("GOV.UK Payment ID")
+    private String transactionId;
+
+    @JsonProperty("Issued By")
+    private String issuedBy;
+
+    @JsonProperty("Date Created")
+    private String dateCreated;
+
+    @JsonProperty("Time Created")
+    private String timeCreated;
+
+    @JsonProperty("Corporate Card Surcharge")
+    private String corporateSurcharge;
+
+    @JsonProperty("Total Amount")
+    private String totalAmount;
+
+    @JsonProperty("Wallet Type")
+    private String walletType;
+
+    @JsonProperty("Card Type")
+    private String cardType;
+
+    public FlatCsvTransaction() {
+
+    }
+
+    public FlatCsvTransaction(
+            String reference, String description, String email, String amount, String cardBrand, String cardholderName,
```

Comment:
A builder would be nice here.

Reply:
Agreed 👍 this can be done in the next PR depending on if this POJO will actually be used going forward.